pyproject.toml

```diff
@@ -0,0 +1,172 @@
+[project]
+name = "nbcollection"
+dynamic = [
+    "version"
+]
+description = "Execute and convert collections of Jupyter notebooks to static HTML sites."
+license = {file = "LICENSE"}
+readme = "README.md"
+authors = [
+    { name = "The Astropy Developers", email = "astropy.team@gmail.com" },
+    { name = "Adrian Price-Whelan", email = "adrianmpw@gmail.com" },
+    { name = "Erik Tollerud" }
+]
+requires-python = ">=3.9"
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD-3-Clause",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+keywords = [
+    "astronomy",
+    "jupyter",
+    "notebooks",
+    "tutorials",
+]
+dependencies = [
+    "nbconvert",
+    "jupyter-client",
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest>=7.0",
+    "notebook",
+]
+
+[project.urls]
+repository = "https://github.com/astropy/nbcollection"
+
+[project.scripts]
+nbcollection = "nbcollection.__main__:main"
+fitscheck = "astropy.io.fits.scripts.fitscheck:main"
+fitsdiff = "astropy.io.fits.scripts.fitsdiff:main"
+fitsheader = "astropy.io.fits.scripts.fitsheader:main"
+fitsinfo = "astropy.io.fits.scripts.fitsinfo:main"
+samp_hub = "astropy.samp.hub_script:hub_script"
+showtable = "astropy.table.scripts.showtable:main"
+volint = "astropy.io.votable.volint:main"
+wcslint = "astropy.wcs.wcslint:main"
```

Comment:
Whoops - were these there before? I don't think they need to be (or should be) there?

Reply:
Ah thanks for catching that. Left-over from copy-pasting from astropy :)